spec/tree_spec.rb

```diff
@@ -50,6 +50,37 @@
     it "has blobs" do
       expect(@tree.blobs).to be_kind_of Array
     end
+
+    it "finds blobs and trees efficiently" do
```

Comment:
Is there a way to find multiple Blobs/Trees at once?

Reply:
If so, it would be good to have a test showcasing that.

Reply:
`#find_all` was implemented by default because of `Enumerable`. But that uses the slow loop over the contents array, so I've now added an efficient `#find_all`.